src/backend/distributed/master/shard_rebalancer.c

```diff
@@ -0,0 +1,28 @@
+/*-------------------------------------------------------------------------
+ *
+ * shard_rebalancer.c
+ *
+ * Function definitions for the shard rebalancer tool.
+ *
+ * Copyright (c) 2019, Citus Data, Inc.
+ *
+ * $Id$
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+#include "fmgr.h"
+
+
+#define NOT_SUPPORTED_IN_COMMUNITY(name) \
```

Comment:
That sees like a useful macro. Is there an appropriate header file?

Reply:
Yeah I was thinking the same, but I don't know the codebase well enough to know where it should be.